src/index.html

```diff
@@ -2,13 +2,134 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge"
+    >
+    <title>
+      Moyo header
+    </title>
+    <link
+    rel="stylesheet"
+    href="./style.css"
+    >
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+    >
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    >
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet"
+    >
```

Comment:
Add 2 spaces everywhere for attributes
```suggestion
    <link
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
      rel="stylesheet"
    >
```